isis/src/base/objs/Camera/Camera.cpp

```diff
@@ -2303,7 +2303,14 @@ namespace Isis {
     if (sin(b) == 0.0 || sin(c) == 0.0) {
       return azimuth;
     }
-    double A = acos((cos(a) - cos(b)*cos(c))/(sin(b)*sin(c))) * 180.0 / PI;
+    double intermitent = (cos(a) - cos(b)*cos(c))/(sin(b)*sin(c));
```

Comment:
Correct spelling is "intermittent."

The word "intermittent" means "sporadic" or "occasional." The mathematical expression simply defines what will become the argument to acos(). Consider using a different name that more closely describes the expression.

Reply:
Updated to `intermediate`. I think that was what I wanted original just crossed wires in my brain

Reply:
No worries, it happens.